cpp/src/fil/infer.cu

```diff
@@ -36,7 +36,7 @@
 #endif  // __CUDA_ARCH__
 #endif  // CUDA_PRAGMA_UNROLL
 
-#define INLINE_CONFIG __forceinline__
+#define INLINE_CONFIG __noinline__
```

Comment:
Was this intended to be reverted before merger?

Reply:
Fixed.

Thanks for catching it!